library/formatting_DateToYYYYMMDD_js.js.php

```diff
@@ -32,8 +32,15 @@ function DateToYYYYMMDD_js(value){
 }
 
 function TimeToHHMMSS_js(value){
-    //For now, just return the Value, since input fields are not formatting time.
-    // This can be upgraded if decided to format input time fields.
+    if (value.trim() == '') {
+        return '';
+    }
+
+    var is_pm = toUpperCase(value.trim()).indexOf('PM');
+    if (is_pm) {
+        let d = new Date("1970-01-01 " + value);
+        let value = d.setHours(d.getHours() + 12).toTimeString();
+    }
```

Comment:
haven't tested this :grimacing: 

looks like would have to run some cqm reports to do so

Reply:
tested it a little bit to get it working for cqm report generation in Reports->Clinic->Automated Measures (AMC)